lib/Minz/Session.php

```diff
@@ -4,18 +4,31 @@
  * La classe Session gère la session utilisateur
  */
 class Minz_Session {
+	private static $volatile = false;
+	
 	/**
 	 * Initialise la session, avec un nom
 	 * Le nom de session est utilisé comme nom pour les cookies et les URLs(i.e. PHPSESSID).
-	 * Il ne doit contenir que des caractères alphanumériques ; il doit être court et descriptif
+	 * Il ne doit contenir que des caractères alphanumériques ; il doit être court et descriptif.
+	 * If the volatile parameter is true, then no cookie and not session storage are used.
+	 * Volatile is especially useful for API calls without cookie / Web session.
 	 */
-	public static function init($name) {
+	public static function init($name, $volatile = false) {
```

Comment:
New concept of volatile session, to avoid a warning with our API calls, and with a nice optimisation that avoid creating local garbage for unneeded session data on disk.